auto_install/install.sh

```diff
@@ -2177,10 +2327,16 @@ confOVPN(){
 
 confWireGuard(){
 	# Reload job type is not yet available in wireguard-tools shipped with Ubuntu 20.04
-	if ! grep -q 'ExecReload' /lib/systemd/system/wg-quick@.service; then
-		echo "::: Adding additional reload job type for wg-quick unit"
-		$SUDO install -D -m 644 "${pivpnFilesDir}"/files/etc/systemd/system/wg-quick@.service.d/override.conf /etc/systemd/system/wg-quick@.service.d/override.conf
-		$SUDO systemctl daemon-reload
+	if [ "${PLAT}" == 'Alpine' ]; then
+		echo '::: Adding wg-quick unit'
+
+		${SUDO} install -m 644 "${pivpnFilesDir}/files/etc/init.d/wg-quick" /etc/init.d/wg-quick
```

Comment:
```suggestion
		${SUDO} install -m 0755 "${pivpnFilesDir}/files/etc/init.d/wg-quick" /etc/init.d/wg-quick
```

Won't start if not executable